src/Core/src/Core/IBorder.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Microsoft.Maui.Graphics;
+
+namespace Microsoft.Maui
+{
+	/// <summary>
+	/// Provides functionality to set the view borders.
+	/// </summary>
+	public interface IBorder
+	{
+		/// <summary>
+		/// Gets the the brush of a view's border.
+		/// </summary>
+		Paint? BorderBrush { get; }
+
+		/// <summary>
+		/// Gets the the width of a view's border.
+		/// </summary>
+		double BorderWidth { get; }
+
+		/// <summary>
+		/// Gets a collection of Double values that indicate the pattern of dashes and gaps 
+		/// that is used to outline shapes.
+		/// </summary>
+		DoubleCollection BorderDashArray { get; }
```

Comment:
Can we not just use double[] and have the converter work with that?  The `DoubleCollection` should stay in Controls, not get moved down to Core.